applications/serial_lte_modem/Kconfig

```diff
@@ -104,15 +98,12 @@ config SLM_INDICATE_TIME
 	  GPIO active indication time in milliseconds. This setting specify the period length for the pin to be active
 
 #
-# Socket
+# LTE network automatic attach
 #
-config SLM_SOCKET_RX_MAX
-	int "Maximum RX buffer size for receiving socket data"
-	range 576 708
-	default 576
+config SLM_AUTO_CONNECT
+	bool "Automatically connect to the LTE network on start-up or after a reset"
 	help
-	  Default: NET_IPV4_MTU (576)
-	  Maximum: MSS setting in modem (708)
+	  Enables automatically connecting to the LTE network on start-up using the default PDN configuration..
```

Comment:
Should likely point to configuration need in here as well. The default PDN configuration that is currently setup serves the LWM2M customers, but not others.

Reply:
Please suggest changes

Reply:
Suggested.